### PR TITLE
Fire damage by rads is log(rads) * 2

### PR DESCRIPTION
**Pull request by yoyobatty**
Mirror of: https://github.com/tgstation/tgstation/pull/46806

Makes it so stupidly high radiation level can't insta kill you but at the same time reduces burn damage from rads. Don't worry, you'll still accumulate toxin damage.

### CODE REVIEW

**Comment by karmaisblackandbluepilled** [comment]
-1 fusion instakills are based

File: html/changelogs/AutoChangeLog-pr-46806.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+author: "Tlaltecuhtli"
```
Comment (ghost) [inline html/changelogs/AutoChangeLog-pr-46806.yml:1]: Won't our bot take care of this for us?

**Review by ghost — APPROVED** [review]

**Comment by ghost** [comment]
@steamp0rt reeeeeeeeeeee don't approve when I've started a review

also, +1. Instakills are shit and should be removed, especially ones this low-effort.

**Review by ghost — APPROVED** [review]
Never mind - since you didn't put logs in the PR, this might as well be the way to go

**Comment by Maroonenahmoron** [comment]
+1, let radiation mess up the person more before killing them.

**Comment by ghost** [comment]
HA approved, mergo time

**Pull request merged**